### ISSUE TITLE
Shoulder increased external rotation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10999]
**Preferred term label:**
Shoulder increased external rotation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased rotation of the lower part of the arm away from the body (tested with the elbow flexed and the elbow held next to the side of the body or shoulder 90 degrees abducted to the side)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shoulder internal impingement syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Not sure I understand. Are you talking about the adaptation that pitchers have, e.g., https://pmc.ncbi.nlm.nih.gov/articles/PMC5155252/ 
That is not really an abnormal finding?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK554518/ says "Shoulder impingement syndrome is a painful condition of the upper extremity resulting from a structural narrowing of the subacromial space. It is primarily diagnosed by history and physical examination. The mainstay of treatment involves identification early before the onset of degenerative changes, physical therapy exercises to strengthen the shoulder girdle, and pharmacologic interventions to decrease inflammation. This activity describes the evaluation and management of shoulder impingement syndrome and highlights the role of an interprofessional team in the care of patients with this condition."

We now also have this finding for:
Pectoralis major rupture
Proximal humeral epiphysiolysis
Rotator cuff tear arthropathy

**Comment by pnrobinson** [comment]
@MickeySegal 
the article you cite writes: " Often, patients will have weakness of abduction and/or external rotation of the affected side", however, it is not clear to me that there is **pathologically increased** external rotation, which seems to be the intent of this new term?

**Comment by MickeySegal** [comment]
My comment above appears to be addressing a different finding.  I marked it as to be ignored.

The correct response is that https://pmc.ncbi.nlm.nih.gov/articles/PMC5155252/ notes in Table 1, "Keys to diagnosing internal impingement" has "Increased external rotation, decreased internal rotation".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001329
- label: Increased external rotation of shoulder
- parents: [Large joint hypermobilty (HP:0430047)](https://hpo.jax.org/app/browse/term/HP:0430047)
- pmid: 
- def: Increased rotation of the lower part of the arm away from the body (tested with the elbow flexed and the elbow held next to the side of the body or shoulder 90 degrees abducted to the side).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10999
- synonyms: Shoulder increased external rotation(NONE)
